CONTRIBUTING.md

```diff
@@ -23,11 +23,13 @@ An issue is either a **bug** (an unexpected / unwanted behavior of the software
 
 ### Filing Bugs
 
-If you file a **bug issue**, please provide at least the following information:
+If you file a **bug issue**, use the following link to [Create a bug report](https://github.com/Scille/parsec-cloud/issues/new?template=bug-report.yml&title=[%F0%9F%90%9B+|+Bug]%3A+).
 
-1. version of Parsec (either by doing `parsec --versions`, by getting it from the GUI, or by providing the git tag if you work from the sources)
-2. error/log output (eg a traceback)
-3. steps to reproduce the issue
+It's a template that list the required info that we need about it, you will need to provide:
+
+1. The version of Parsec (either by doing `parsec --versions`, by getting it from the GUI, or by providing the git tag if you work from the sources).
+2. Error/log output (eg a traceback).
+3. Steps to reproduce the issue.
```

Comment:
I think this is redundant with the report itself. 
BTW, the instructions to obtain Parsec version should be moved to the bug report.

Reply:
```suggestion
```